### PR TITLE
implements the new TOC to non time series plot types

### PR DESCRIPTION
**Pull request by FBRTMaka**
@Bobfrat , @birdage , Just wondering about the depth profile, in the test it's looking for two but it's set to only render one.  Compared with with Casadev and it's seems like it'll never show (since you can't choose two variables).

### CODE REVIEW

**Comment by birdage** [comment]
@FBRTMaka shouldnt we use `.get` over the explicit attribute grab?

File: ooiui/static/js/views/common/SVGView.js [hunk 1]
```diff
@@ -130,15 +133,18 @@ var plotParameters ={
...
       return this._selectedX._validateNumberOfSelected("scatter", yvar)
-    }else if ($.inArray(options.plotType, check2yvarplots)!=-1) {
+    }else if ($.inArray(options.attributes.data.plotType, check2yvarplots)!=-1) {
       var var1 = $("#yvar1-select :selected").text();
       var var2 = $("#yvar2-select :selected").text();
       var yvar = [];
       if ( var1 != 'undefined' ) yvar.push(var1);
       if ( var2 != 'undefined' ) yvar.push(var2);
-      return this._selectedX._validateNumberOfSelected(options.plotType, yvar)
+      return this._selectedX._validateNumberOfSelected(options.attributes.data.plotType, yvar)
```
Comment (birdage) [inline ooiui/static/js/views/common/SVGView.js]: @FBRTMaka example, is options a model?

File: ooiui/static/js/views/common/SVGView.js [hunk 2]
```diff
@@ -227,56 +233,9 @@ var SVGView = Backbone.View.extend({
 });
 
 var SVGPlotView = SVGView.extend({
-  setModel: function(model) {
-    this.model = model;
-    this.reference_designator = this.model.get('reference_designator')
-    this.stream_name = this.model.get('stream_name')
-    this.variable = this.model.get('yvariable').join()
-    this.dpa_flag = "0";
-    /* NEED TO ADD THIS BACK
```
Comment (birdage) [inline ooiui/static/js/views/common/SVGView.js]: @FBRTMaka `  /* NEED TO ADD THIS BACK`.....?

File: ooiui/templates/asset_management/assetslist.html [hunk 3]
```diff
@@ -35,7 +35,7 @@
             <div id="assetTableHeading" class="panel-heading hidden">
                 <!-- ooiui/static/js/partials/AssetTableHeader.html -->
             </div>
-            <i class="fa fa-circle-o-notch fa-spin" style="margin-left:50%; font-size:90px;"></i>
+            <i class="fa fa-spinner fa-spin" style="margin-left:50%; font-size:90px;"></i>
```
Comment (birdage) [inline ooiui/templates/asset_management/assetslist.html]: @FBRTMaka :+1:

**Comment by FBRTMaka** [comment]
@birdage , since a model isn't actually being passed around anymore, can't use .get.  it did work when the model was created by the view and then passed around...but I can't bring myself to keep that MVC blaspheme lol

SO, the way it's implemented .get isn't actually a function anymore.

If we can get to the model and include many of the attributes up front, THEN pass the model around it would be a lot cleaner.  

"_smart_ models, _thin_ controllers, _dumb_ views "

**Comment by birdage** [comment]
:+1:

File: ooiui/static/js/views/common/SVGView.js (hunk 2 above)
Comment (FBRTMaka) [inline ooiui/static/js/views/common/SVGView.js]: @birdage  not sure who put that in there but it's working fine without it, it's been committed out for ages!

**Pull request merged**